ui/pages/swaps/prepare-swap-page/review-quote.js

```diff
@@ -1045,6 +1078,17 @@ export default function ReviewQuote({ setReceiveToAmount }) {
     }
   };
 
+  let tradeTxTokenFee;
+  if (isGasIncludedTrade) {
+    tradeTxTokenFee =
+      smartTransactionFees?.tradeTxFees?.fees?.[0]?.tokenFees?.[0] || null;
+  }
```

Comment:
The `let` isn't needed.

```suggestion
  const tradeTxTokenFee = isGasIncludedTrade
    ? smartTransactionFees?.tradeTxFees?.fees?.[0]?.tokenFees?.[0] ?? null
    : null;
```